www/class/centreon-knowledge/procedures_Proxy.class.php

```diff
@@ -2,7 +2,7 @@
 /*
  * MERETHIS
  *
- * Source Copyright 2005-2010 MERETHIS
+ * Source Copyright 2005-2018 MERETHIS

```

Comment:
That's the kind of licence for wich I need the advise of JMA and LPI...
Should we replace MERETHIS with CENTREON ? I don't know

Reply:
I believe we need create a new line with a range of years, example:
```
Source Copyright 2005-2010 MERETHIS
Source Copyright 2011-2018 CENTREON
```